<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InscriptionElecteurClass</class>
 <widget class="QDialog" name="InscriptionElecteurClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InscriptionElecteur</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Electeur_Prenom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Electeur_Ville">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Electeur_Nom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Province:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero civic:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Electeur_Nas">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Electeur_Province">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Electeur">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Electeur_CodePostal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Electeur_Ok">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Electeur_NumeroCivic">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Electeur_NomRue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom rue:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inscription dun electeur</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Electeur_Ok</sender>
   <signal>clicked()</signal>
   <receiver>InscriptionElecteurClass</receiver>
   <slot>validerInscriptionElecteur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerInscriptionElecteur()</slot>
 </slots>
</ui>
